backend/svirt.pm

```diff
@@ -174,6 +175,30 @@ sub load_snapshot ($self, $args) {
     return $post_load_snapshot_command;
 }
 
+sub do_extract_assets ($self, $args) {
+    my $first_hdd = $bmwqemu::vars{S390_ZKVM} ? 'a' : 'b';
+    my $name = $args->{name};
+    die "No name given" unless $args->{name};
+    $name =~ /\.([[:alnum:]]+)$/;
+    die "No HDD specified" unless $args->{hdd_num};
+    my $hdd_num = $args->{hdd_num} - 1;
+    die "No format specified" unless $args->{format};
+    my $format = $args->{format};
+    return undef if (($format ne 'raw') and ($format ne 'qcow2'));
+    my $svirt = $self->select_console('svirt');
+    my $svirt_name = $svirt->name . chr(ord($first_hdd) + $hdd_num);
+
+    my $image_storage = '/var/lib/libvirt/images';
```

Comment:
I think the asset has to be stored in `$args->{dir}` to be picked up

Reply:
Sorry, I meant that the final file needs to be in `$args->{dir}`

Reply:
We have `IMAGE_STORAGE` defined in the backend now.